content/en/testing-guide/_index.adoc

```diff
@@ -1,65 +1,26 @@
-image::https://photos.smugmug.com/Tetra-Testing/PAPRa-Build-13-March-2021/i-N5dwqjg/0/a18cf5f4/L/_MMR0431-L.jpg[] 
-
-*NOTE:* The Portacount is effective at showing the isolation of airborne particles, but not at identifying the sizes of particles measured, and did not have a setting for an N100 level of protection.  When testing for N95 levels of protection, the Portacount has a range from 0 to 200+, with 100 passing, while N99 testing has a range from 0 to 2000+, with 500 passing.  All tests depicted in this document were done using online testing, waiting several seconds to ensure that the readings were stable before recording them.
-
-== Testing Procedure
-
-Prior to testing, a saline solution was aerosolized to ensure that there would be a sufficient quantity of airborne particles to demonstrate effective filtration.  
-
-[link=https://photos.smugmug.com/Tetra-Testing/PAPRa-Build-13-March-2021/i-h594S5t/0/ab2e7757/5K/_MMR0404-5K.jpg]
-image::https://photos.smugmug.com/Tetra-Testing/PAPRa-Build-13-March-2021/i-h594S5t/0/ab2e7757/L/_MMR0404-L.jpg[] 
-
-[link=https://photos.smugmug.com/Tetra-Testing/PAPRa-Build-13-March-2021/i-vJ5rXgW/0/ea8ea9ed/5K/_MMR0405-5K.jpg]
-image::https://photos.smugmug.com/Tetra-Testing/PAPRa-Build-13-March-2021/i-vJ5rXgW/0/ea8ea9ed/L/_MMR0405-L.jpg[] 
-
-[link=https://photos.smugmug.com/Tetra-Testing/PAPRa-Build-13-March-2021/i-rwBPpgs/0/e1b9d481/5K/_MMR0414-5K.jpg]
-image::https://photos.smugmug.com/Tetra-Testing/PAPRa-Build-13-March-2021/i-rwBPpgs/0/e1b9d481/L/_MMR0414-L.jpg[] 
-
-The Portacount measurement hose was attached to the port in the hose array depicted below:
-
-[link=https://photos.smugmug.com/Tetra-Testing/PAPRa-Build-13-March-2021/i-jjzRxbT/0/5267278f/5K/_MMR0424-5K.jpg]
-image::https://photos.smugmug.com/Tetra-Testing/PAPRa-Build-13-March-2021/i-jjzRxbT/0/5267278f/L/_MMR0424-L.jpg[] 
-
-The other hose, the blue hose, is left exposed to the air to provide a sample of the ambient air to the Portacount.
-
-Here are the videos of the various tests performed:
-
-video::https://photos.smugmug.com/Tetra-Testing/PAPRa-Build-13-March-2021/i-Kq8Lp8G/0/37bf76f3/1920/IMG_5191-1920.mp4[]
+We came up with a series of tests to ensure that the PAPRa would be performant.  Performant means:
 
-Testing the fit of the mask itself:
+1.  When worn, the wearer received N99 levels of protection. link:testing_march_2021[This page] describes how we did that testing in March of 2021 on both individual fan units and on a tester wearing the entire PAPRa.
+2.  The unit must provide a minimum amount of flow.  Hooded PAPRs require a flow of 6 CFM.  Since the PAPRa is designed to provide airflow for a half facepiece, the flow requirement is 4 CFM, or 113.27 L/min.  As of link:flow_testing_may_2021[May 2021], PAPRas provide 67-92 L/min of airflow, depending on hose configuration, so more work is necessary to bring the units up to this flow specification[1]. 
```

Comment:
Specs should be reviewed by a subject matter expert

Reply:
Not sure what we need here-- ask Daniel, maybe?  He's told me that these are reasonable goals.

Reply:
Will mark it as resolved then